test/exercise/arrays/solution.rb

```diff
@@ -1,12 +1,38 @@
 module Exercise
   module Arrays
     class << self
+      def max(array)
+        array.reduce { |acc, current | acc > current ? acc : current }
+      end
+
       def replace(array)
-        array
+        maximum = max(array)
+        array.map { |number| number > 0 ? maximum : number }
```

Comment:
I believe, rubocop would suggest using `number.positive?` instead of a comparison. Have you ran it?

Reply:
fixed, rubocop did not complain about this line, anyway I've used `number.positive?`